dist/components/Button/Button.test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
I think we can exclude tests from the `dist` folder right? users won't need them